builtin/game/chat.lua

```diff
@@ -1028,71 +1074,70 @@ core.register_chatcommand("clearobjects", {
 				.. options.mode .. " mode).")
-		core.chat_send_all("Clearing all objects. This may take a long time."
-				.. " You may experience a timeout. (by "
-				.. name .. ")")
+		core.chat_send_all(S("Clearing all objects. This may take a long time. "
+			.. "You may experience a timeout. (by @1)", name))
 		core.clear_objects(options)
 		core.log("action", "Object clearing done.")
-		core.chat_send_all("*** Cleared all objects.")
+		core.chat_send_all("*** "..S("Cleared all objects."))
 		return true
 	end,
 })
 
 core.register_chatcommand("msg", {
-	params = "<name> <message>",
-	description = "Send a direct message to a player",
+	params = S("<name> <message>"),
+	description = S("Send a direct message to a player"),
 	privs = {shout=true},
 	func = function(name, param)
 		local sendto, message = param:match("^(%S+)%s(.+)$")
 		if not sendto then
-			return false, "Invalid usage, see /help msg."
+			return false, S("Invalid usage, see /help msg.")
 		end
 		if not core.get_player_by_name(sendto) then
-			return false, "The player " .. sendto
-					.. " is not online."
+			return false, S("The player @1 is not online.", sendto)
 		end
 		core.log("action", "DM from " .. name .. " to " .. sendto
 				.. ": " .. message)
-		core.chat_send_player(sendto, "DM from " .. name .. ": "
-				.. message)
-		return true, "Message sent."
+		core.chat_send_player(sendto, S("DM from @1: @2", name, message))
+		return true, S("Message sent.")
 	end,
 })
 
 core.register_chatcommand("last-login", {
-	params = "[<name>]",
-	description = "Get the last login time of a player or yourself",
+	params = S("[<name>]"),
+	description = S("Get the last login time of a player or yourself"),
 	func = function(name, param)
 		if param == "" then
 			param = name
 		end
 		local pauth = core.get_auth_handler().get_auth(param)
 		if pauth and pauth.last_login and pauth.last_login ~= -1 then
 			-- Time in UTC, ISO 8601 format
-			return true, param.."'s last login time was " ..
-				os.date("!%Y-%m-%dT%H:%M:%SZ", pauth.last_login)
+			return true, S("@1's last login time was @2.",
+				param,
+				os.date("!%Y-%m-%dT%H:%M:%SZ", pauth.last_login))
 		end
-		return false, param.."'s last login time is unknown"
+		return false, S("@1's last login time is unknown.", param)
 	end,
 })
 
 core.register_chatcommand("clearinv", {
-	params = "[<name>]",
-	description = "Clear the inventory of yourself or another player",
+	params = S("[<name>]"),
+	description = S("Clear the inventory of yourself or another player"),
 	func = function(name, param)
 		local player
 		if param and param ~= "" and param ~= name then
 			if not core.check_player_privs(name, {server=true}) then
-				return false, "You don't have permission"
-						.. " to clear another player's inventory (missing privilege: server)"
+				return false, S("You don't have permission to "
+					.. "clear another player's inventory "
+					.. "(missing privilege: server).")
```

Comment:
```suggestion
					.. "(missing privilege: @1).", "server")
```
for same reasons as mentioned above.